src/main/java/net/mcreator/plugin/PluginLoader.java

```diff
@@ -184,4 +188,26 @@ public PluginLoader() {
 		return plugin;
 	}
 
+	public void checkPluginUpdates() {

```

Comment:
This one should be private

Reply:
I decided to put it public because it will, later, allow us to add a button in MCreator to check for updates, a bit like you did for MCreator updates.

Reply:
Still it should be loaded once only, and then only queried, so each call does not invoke full parsing and read